pipeline/transform.py

```diff
@@ -1,3 +1,125 @@
+"""Transforms the data by flattening and cleaning before it goes into the load stage"""
 
+import pandas as pd
+from pandas import DataFrame
+
+
+def get_rss_feed_df(file_path: str) -> DataFrame:
+    """Returns the scraped article csv as a DataFrame"""
+
+    df = pd.read_csv(file_path)
+
+    return df
+
+
+def get_scraped_articles_df(file_path: str) -> DataFrame:
+    """Returns the rss_feed csv as a DataFrame"""
+
+    df = pd.read_csv(file_path)
+
+    return df
+
+
+def format_time_to_timestamp(time_in_col: str) -> str:
```

Comment:
have you tried datetime instead of this function?

Reply:
datetime.strptime()

Reply:
Reworked :)